llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9485,44 +9494,6 @@ void VPWidenMemoryInstructionRecipe::execute(VPTransformState &State) {
     }
```

Comment:
nit: "only neede[d] for" typo above.

Reply:
Fixed in 516cc98affa509614f1f80dc8870e8ec6183e6c7